### ISSUE TITLE
Označiť vstupy v protokole

### ISSUE THREAD

**Issue by david58** [trojsten/ksp-school#28]
![2023-07-06-193230_505x616_scrot](https://github.com/trojsten/ksp-school/assets/11067153/8661fd0a-2b4f-49d1-8334-4a317d1d9c4d)

ten diff s názvom vstupu nedáve veľký zmzsel pri tom náhodnom počte farebných štvorčekov :D 

ak tomu správne rozumiem tak delenie na sady v school nedáva veľký zmysel, tak by bolo pekné ak by tie štvorčeky mali nejaký konzistentný formát a aby sa dalo nejako zistiť  ako sa volá vstup na ktorom submit padá

na diskusiu: pri testovači sme sa viac krát bavili či by nemalo zmysel ukazovať diff pre všetky vstupy aj userom, nie len adminom. Na rozdiel od súťaží tá informácia iba pomáha, a nemusí potom zakaždým hladať ten submit vedúci a pozerať sa aký bol diff

**Comment by marcel3032** [comment]
Keď nad vstup hoverneš, nezobrazí sa? Ale áno, o tom sme sa niekedy rozprávali.

**Comment by marcel3032** [comment]
Ako by si si predstavoval delenie na sady?

**Comment by marcel3032** [comment]
A teda diff je defaultne skrytý, dá sa zobraziť. Tiež som sa nad tým zamýšľal, ale trochu bojujem s tým, že často ti samotný diff veľa nepovie.. že ideálne by si proste chcel mať aj vstupné dáta

**Comment by marcel3032** [comment]
No, ono zase tie sady často majú zmysel, lebo je asi fajn, že vidíš, že ti to zbehlo na týchto sadách úplne a na týchto ti to vytlelo..

**Comment by marcel3032** [comment]
A teda ak sme takí, že chceme mať viditeľný diff, tak v pohodr prepíšem ten jeden `False` na `True` 😁

**Comment by Aiq0** [comment]
protokol uz renderuje judge a aj viditelnost diffov sa v nom tiez da nastavit, cize ak je o to zaujem, da sa to prestavit
